native/cocos/editor-support/spine/Extension.cpp

```diff
@@ -47,7 +47,9 @@ void SpineExtension::setInstance(SpineExtension *inValue) {
 }
 
 SpineExtension *SpineExtension::getInstance() {
+#ifndef __EMSCRIPTEN__
     if (!_instance) _instance = spine::getDefaultExtension();
+#endif
```

Comment:
Why need to change it?

Reply:
Since doesn't need getDefaultExtension. It's a singleton and always lives in the whole lifetime.
It's really redutant. You could refer to the code:
```ts
void SpineWasmUtil::spineWasmInit() {
    // LogUtil::Initialize();
    SpineExtension* extension = new WasmSpineExtension();
    SpineExtension::setInstance(extension); // Set the singleton here.

    SpineMeshData::initMeshMemory();

    //LogUtil::PrintToJs("spineWasmInit");
}

void SpineWasmUtil::spineWasmDestroy() {
    auto* extension = SpineExtension::getInstance();
    delete extension;
    freeStoreMemory();
    SpineMeshData::releaseMeshMemory();
    // LogUtil::ReleaseBuffer();
}
```

Remove `getDefaultExtension()` invocation will save some code size.

Reply:
Got it.